assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -102,9 +191,17 @@ const mapDispatchToProps = dispatch => ({
   moveModuleDown: id => dispatch(moveLandingPageModuleDown(id)),
   moveModuleUp: id => dispatch(moveLandingPageModuleUp(id)),
   toggleModule: id => dispatch(toggleLandingPageModule(id)),
-  createModule: (nextOrder, identifier = 'INTRODUCTION') => {
+  createModule: (nextOrder, numberOfDuplicatesModules, identifier = MODULES_IDENTIFIERS.introduction) => {
     const newId = createRandomId();
-    return dispatch(createLandingPageModules(newId, identifier, nextOrder));
+    return dispatch(
+      createLandingPageModules(
+        newId,
+        identifier,
+        numberOfDuplicatesModules,
```

Comment:
`numberOfDuplicates`?
If possible, it is better to add the number here before to call the redux action (to avoid an additional param). You also need to check if there is no duplicates and don't add the suffix in this case.
Something like `I18n.t('administration.landingPage.manageModules.textAndMultimedia', { count: numberOfDuplicates })` should do the trick (and you can even add 'administration.landingPage.manageModules.textAndMultimedia_0` to handle the no duplicates case

Reply:
But it's impossible not to have a duplicate if you are creating an additional module.